### PR TITLE
feat: merge login and register

### LINKED ISSUE #49: Slå sammen "Logg Inn" og "Registrer"

### PR DESCRIPTION
**Pull request by matsjs**
Closes #49

### CODE REVIEW

File: nodebb-theme-fdk/languages/en-GB/fdk.json [hunk 1]
```diff
@@ -38,5 +38,8 @@
   },
   "post": {
     "reply": "Reply to"
+  },
+  "navbar": {
+    "login": "Log inn/ Register"
```
Comment (azaxarov) [inline nodebb-theme-fdk/languages/en-GB/fdk.json:43]:
```suggestion
    "login": "Log in / Register"
```

File: nodebb-theme-fdk/languages/en-US/fdk.json [hunk 2]
```diff
@@ -38,5 +38,8 @@
   },
   "post": {
     "reply": "Reply to"
+  },
+  "navbar": {
+    "login": "Log inn/ Register"
```
Comment (azaxarov) [inline nodebb-theme-fdk/languages/en-US/fdk.json:43]:
```suggestion
    "login": "Log in / Register"
```

File: nodebb-theme-fdk/languages/nb/fdk.json [hunk 3]
```diff
@@ -38,5 +38,8 @@
   },
   "post": {
     "reply": "Svar til"
+  },
+  "navbar": {
+    "login": "Logg inn/ Registrer"
```
Comment (azaxarov) [inline nodebb-theme-fdk/languages/nb/fdk.json:43]:
```suggestion
    "login": "Logg inn / Registrer"
```

**Review by azaxarov — APPROVED** [review]

File: nodebb-theme-fdk/languages/en-GB/fdk.json (hunk 1 above)
Comment (matsjs) [reply nodebb-theme-fdk/languages/en-GB/fdk.json:43]: @azaxarov oops

**Pull request merged**